string name
float64 value
int32 priority
float32 deadline
